dir.props

```diff
@@ -559,6 +559,7 @@
     <PackagesBasePath Condition="'$(PackagesBasePath)'==''">$(BinDir)$(OSPlatformConfig)</PackagesBasePath>
     <PackageOutputPath Condition="'$(PackageOutputPath)'==''">$(PackageOutputRoot)$(ConfigurationGroup)/</PackageOutputPath>
     <SymbolPackageOutputPath Condition="'$(SymbolPackageOutputPath)'==''">$(PackageOutputPath)symbols/</SymbolPackageOutputPath>
+    <LayoutBasePath>$(BaseOutputPath)/layout/</LayoutBasePath>
```

Comment:
Right now it is just the layout directory we are producing with layout.proj but my hope is that it will become our new layout. Although I don't expect it will be PackageLayoutDir as that isn't going to be the full ref set.